tests/behat/features/capabilities/search/search-content.feature

```diff
@@ -35,7 +46,20 @@ Feature: Search
     And I should see "Event one" in the "Main content"
     And I should see "Topic one"
     And I should not see "Event second"
-    # Scenario: Successfully filter search results
+
+    # Test the pager.
+    When I am on "search/content"
+    And I fill in the following:
+      | search_input | Shenanigans |
+    And I press "Search"
+    And I click the xth "0" element with the css ".pager-nav .pager__item--next"
+    And I should see "Topic thirteen"
+    And I fill in the following:
+      | search_input | four |
+    And I press "Search"
+    And I should see "Topic four"
```

Comment:
And here we are verifying that we're back on Page 1 by searching for a single topic which will not generate multiple pages.